src/content/recipes/next.md

```diff
@@ -69,6 +68,31 @@ export default {
 };
 ```
 
+#### (Experimental) SWC support
+
+If you're working with a Next.js project that already uses SWC (e.g., version 13 or higher), you can enable it in Storybook by adding the following to your `main.js`:
```

Comment:
We only support swc mode in Next.js >= 14

Reply:
That's right, thanks for the callout 🙏